packages/repo-collector-types/src/index.ts

```diff
@@ -125,6 +137,9 @@ export interface WebappMetricDataRepoDatapoint {
       browseUrl: string
     }[]
   }
+  sonarCloud?: {
+    testCoverage?: number
```

Comment:
There are one of three things I think we want to signal in the webapp for each GitHub repository:
1. SonarCloud has been set up for the repository, and test coverage is set up (`sonarCloud` is defined and `measures` contains one object with a metric type of `coverage`)
2. SonarCloud has been set up for the repository, but test coverage is missing (`sonarCloud` is defined and `measures` contains zero objects with a metric of type `coverage`)
3. SonarCloud has not been set up for the repository (the response was a 404 and `sonarCloud` is undefined)

Should we explicitly add some fields to the `WebappMetricDataRepoDatapoint` that makes it easier to distinguish between these fields in the webapp? E.g.,
```typescript
sonarCloud: {
  enabled: boolean // false means that no project has been set up for the GitHub repository
  testCoverage?: string // undefined means that no test coverage has been set up
}
```